src/components/Header/Header.component.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {
+  Avatar,
+  ContainerHamburguer,
+  Form,
+  HeaderContainer,
+  HeaderItem,
+  HeaderItemRigth,
+  LabelToggle,
+  MenuButton,
+  Search,
+  SpanHamburguer,
+  Toggle,
+} from './Header.styled';
+import avatar from '../../img/avatar.png';
+
+const Header = () => {
+  return (
+    <HeaderContainer>
+      <HeaderItem>
+
+        {/* hamburguer menu component */}
+        <MenuButton type="checkbox"  data-testid="hamburguerMenu" disabled/>
+        <ContainerHamburguer>
+          <SpanHamburguer></SpanHamburguer>
+          <SpanHamburguer></SpanHamburguer>
+          <SpanHamburguer></SpanHamburguer>
+        </ContainerHamburguer>
+
+        {/* form to search videos */}
```

Comment:
It's ok to use comments, 

There are cases where are necessary, on this case, I think that if you rename the `Fom` component as `SearchVideosForm`, it is self explained and therefore, the comment is not needed anymore.

The same applies for other places where some renames can communicate the purpose of our code instead of comments 😄

Reply:
Or you can create a Sub Component like "SearchVideoForm" and inside, render the `Form` and ´Search´ components